internal/pkg/service/stream/keboolasink/bridge/file.go

```diff
@@ -89,9 +95,16 @@ func (b *Bridge) createStagingFile(ctx context.Context, now time.Time, sink defi
 
 	// Save credentials to database
 	ctx = ctxattr.ContextWith(ctx, attribute.String("file.resourceID", stagingFile.FileID.String()))
-	entity := keboolasink.FileUploadCredentials{SinkKey: file.SinkKey, FileUploadCredentials: *stagingFile}
+	entity := keboolasink.File{
+		SinkKey: file.SinkKey,
```

Comment:
Entity is probably not the best choice for a variable name, especially when the code is longer.

Maybe you can use `keboolaFile` ?